packages/mendel-middleware/index.js

```diff
@@ -277,13 +286,20 @@ function indexedDeps(mods) {
 
 ****/
 
-function notFound(res, error) {
+function logError(code, hash, bundle) {
+    console.log(['MendelError=' + code, 'hash=' + hash, 'bundle=' + bundle].join(' '));
+}
+
+function notFound(res, error, params) {
     var message = "Mendel: ";
     if (!error) {
         message += 'Bundle not found';
     } else {
         message += error.code + ' - ' + error.message;
     }
+
+    logError(error ? error.code : 'UNKNOWN', params.hash, params.bundle);
```

Comment:
I wanted to log errorCode along with hash and bundle. We may do more logging from other functions in this file. Hence, I moved it to a separate function.